pipenv/vendor/vendor.txt

```diff
@@ -21,20 +21,20 @@ pipdeptree==0.13.0
 pipreqs==0.4.9
     docopt==0.6.2
     yarg==0.1.9
-pythonfinder==1.1.7
+pythonfinder==1.1.8
 requests==2.20.0
     chardet==3.0.4
     idna==2.7
     urllib3==1.24
     certifi==2018.10.15
-requirementslib==1.2.5
+requirementslib==1.3.0
     attrs==18.2.0
     distlib==0.2.8
     packaging==18.0
     pyparsing==2.2.2
     pytoml==0.1.19
     plette==0.2.2
-        tomlkit==0.4.6
+        tomlkit==0.5.2
```

Comment:
Don't update this line or otherwise vendoring build will fail.

Reply:
I undid this on the other branch since I need that to pass first anyway